src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx

```diff
@@ -4,24 +4,29 @@ import MeteredFeature from 'components/shared/MeteredFeature';
 import ConfigurableFeature from 'components/shared/ConfigurableFeature';
 import PlanCard from 'components/shared/PlanCard';
 import CostDisplay from 'components/shared/CostDisplay';
+import SkeletonPlanSelector from 'components/shared/SkeletonPlanSelector';
+import { FeatureMap } from 'utils/plan';
```

Comment:
Did a bit of plan refactoring while i was at it

Reply:
Do these absolute imports work when built? I was thinking they broke when you consumed this in TypeScript and that everything needed to be relative

Reply:
the CI all passed so many it's been fixed 🤷‍♂️ I'll check it out

Reply:
Yeah IIRC it works in JS, but in a TS app (React + TS) it blows up. But worth testing. May only be for certain things like the generated GraphQL types